### DISCUSSION TITLE
Customize address_google Column and save values into multiple rows in DB

### DISCUSSION THREAD

**Discussion by sandeepchowdary7** [Laravel-Backpack/community-forum#273] [Q&A (Help)] [answered]
Hello guys,

Here is the form field I'm using in my project. so using Google Maps API and auto-filing the address. so now I would like to save these address values into multiple rows in the database. i.e. city -> city DB column, zipcode -> zipcode DB column, state -> state DB column etc.. Any inputs would be great. Thanks!

```
[
    'name' => 'address',
    'label' => 'Address',
    'type' => 'address_google',
    'tab' => 'Primary Details',
    'store_as_json' => true
]
```

**Comment by tabacitu** [comment]
Hey @sandeepchowdary7 ,

Personally I'd use a [Laravel Mutator](https://laravel.com/docs/8.x/eloquent-mutators) for that. In my Model I'd have something like:

```php
public function setAddressAttribute($value) {
   $this->city = $value['city'];
   $this->street = $value['street'];
   // etc
}
```

Hope it helps!

**Comment by sandeepchowdary7** [comment]
Hello,

I was doing like this but no luck

//Controller
```
[
    'name' => 'address',
    'label' => 'Address',
    'type' => 'address_google',
    'tab' => 'Primary Details',
    'store_as_json' => true
]
```

//Model
```
 public function setStreetAttribute($value) {
      dd($value);
      $this->city = $value['city'];
      $this->state = $value['state'];
      // etc
  }
```

any help why it is not working in this case?

**Comment by sandeepchowdary7** [comment, marked as answer]
woked! Thanks!
